src/Block.php

```diff
@@ -64,25 +113,39 @@ public function getImageUrl() {
     return FALSE;
   }
 
+  /**
+   * HTML Renders the block.
+   *
+   * If google analytics tracking is enabled, the corresponding JavaScript files
+   * are added.
+   *
+   * @return array
+   *   A renderable array with
+   *   `#theme`: set to `links` and
+   *   `#links`: set to `channelLinks()` output, with each link wrapped in a
+   *             '<span>' tag.
+   */
   public function render() {
     drupal_alter('share_light_options', $this->options);
 
-    // add tracking for GA if googleanalytics module is enabled
+    // Add tracking for GA if googleanalytics module is enabled
     // and share tracking is enabled (default: enabled)
     $tracking_enabled = module_exists('googleanalytics') && variable_get('share_light_tracking_enabled', 1);
     if ($tracking_enabled) {
       drupal_add_js(drupal_get_path('module', 'share_light') . '/tracking.js');
-      drupal_add_js(array('share_light' => array(
-        'share_url' => $this->getUrl(FALSE),
-      )), 'setting');
+      drupal_add_js(array(
+        'share_light' => array(
+          'share_url' => $this->getUrl(FALSE),
+        ),
+      ), 'setting');
     }
 
+    drupal_add_js(drupal_get_path('module', 'share_light') . '/hide_mobile_buttons.js');
     $links = $this->channelLinks();
 
     foreach ($links as &$x) {
       $x['title'] = "<span>{$x['title']}</span>";
       $x['html'] = TRUE;
-      $x['attributes']['target'] = '_blank';
```

Comment:
`target` moved to each Channel separately as mobile buttons do not need `target=_blank`